Update all examples (#557)
effector/effector: docs/recipes/react/example-forms.md

  target: sendFormFx,
})

const Field = ({name, type, label}) => {
  const value = useStoreMap({
    store: $form,
    keys: [name],

>     fn: values => values[name] ?? '',

Reviewer: ```suggestion
    fn: (values, [name]) => values[name] ?? '',
```

  )
}

const App = () => {

>   const handleSubmit = e => {
>     e.preventDefault()
>   }

Reviewer 2: ```suggestion
  const handleSubmit = e => {
    e.preventDefault()
    submitted()
  }
```

And, finally, the `App` itself! Note, how we got rid of any business-logic in view layer. It's simpler to debug, to share logic, and even more: logic is framework independent now.

```js

> const App = () => {
>   const handleSubmit = e => {
>     e.preventDefault()
>   }
>   
>   return (
>     <form onSubmit={handleSubmit}>
>       <Field name="login" label="Login" />
>       <Field name="password" type="password" label="Password" />
>       <button type="submit">Submit!</button>
>     </form>
>   )

Reviewer 2: ```suggestion
Prevent default html form submit behavior using react event from `submitted`

submitted.watch(e => {
  e.preventDefault()
})
const App = () => {
  return (
    <form onSubmit={submitted /*note, there is an event, which is clock for sample*/}>
      <Field name="login" label="Login" />
      <Field name="password" type="password" label="Password" />
      <button type="submit">Submit!</button>
    </form>
  )
```

  </button>
))

const App = () => {
  const handleSubmit = e => {
    e.preventDefault()
    onSubmit()
  }

  return (
    <form onSubmit={onSubmit}>
      Login: <input name="name" />
      <br />
      Password: <input name="password" type="password" />
      <br />
      <Loader />
      <SubmitButton />
    </form>
  )

> }

Reviewer 2: Please pay attention for such changes, example should not become broken even if it was looking weird

```jsx
//transforming upcoming data, from DOM Event to FormData
const onSubmit = sendFormFx.prepend(e => new FormData(e.target)) 

onSubmit.watch(e => {
  e.preventDefault()
})

const App = () => {
  return (
    <form onSubmit={onSubmit}>
      Login: <input name="name" />
      <br />
      Password: <input name="password" type="password" />
      <br />
      <Loader />
      <SubmitButton />
    </form>
  )
}
```

If you don't like to call `preventDefault` from `watch`, then you might replace that with a better example
